app/controllers/apps_controller.rb

```diff
@@ -10,6 +10,7 @@ def update
 
     @app.maintenance = params['maintenance'] if params['maintenance']
     @app.dependencies = params['dependencies'] if params['dependencies']
+    @app.whitelist = params['whitelist'].split(',').map { |s| s.strip }.compact if params['whitelist']
```

Comment:
Ruby has a cool shorthand for when you're looping over something (map/each) and just want to call a single method on each argument:

`@app.whitelist = params['whitelist'].split(',').map(&:strip).compact if params['whitelist']`